public/app/features/integrated-alerting/components/AlertRules/AddAlertRuleModal/AddAlertRuleModal.tsx

```diff
@@ -111,7 +112,12 @@ export const AddAlertRuleModal: FC<AddAlertRuleModalProps> = ({ isVisible, setVi
 
             <TextInputField label={Messages.thresholdField} name="threshold" />
 
-            <NumberInputField label={Messages.durationField} name="duration" validators={[required]} />
+            <NumberInputField
+              inputProps={{ min: MINIMUM_DURATION_VALUE }}
+              label={Messages.durationField}
+              name="duration"
+              validators={[required, minValidator(MINIMUM_DURATION_VALUE)]}
```

Comment:
Consider taking the validators array out of the component to avoid re-creation on every render.

Reply:
👍